lib/improver/wxcode/wxcode_utilities.py

```diff
@@ -78,6 +78,13 @@ def add_wxcode_metadata(cube):
     cube.standard_name = None
     cube.var_name = None
     cube.units = "1"
-    cube.attributes.update({'weather_code': WXCODE})
-    cube.attributes.update({'weather_code_meaning': WXMEANING})
+    wx_keys = np.array(sorted(WX_DICT.keys()))
+    cube.attributes.update({'weather_code': wx_keys})
+    wxstring = ''
```

Comment:
I think this step of putting a list of strings into one string can be simplified by:
```python
wxstring = " ".join(WXDICT.values())
cube.attributes.update({'weather_code_meaning': wxstring})
```